### DISCUSSION TITLE
Create user-defined coordinate system

### DISCUSSION THREAD

**Discussion by DannieCW** [Dlubal-Software/RFEM_Python_Client#276] [Q&A] [answered]
Dear developers, 

I would like to apply a polygon/line load with a direction that is rotated around the Global Z axis by an angle parameter. 
FreeLoad.PolygonLoad or LineLoad.Force cannot be specified with an angle. 
I therefore would like to create a rotated coordinate system, is this achievable via API?
Or could you suggest another way to achieve the first line? 

Thanks. 

Best regards,
Dannie

**Comment by OndraMichal** [comment, marked as answer]
Hi @DannieCW,
create new coordinate system with help of set_coordinate_system function feeding definition of coordinate_system object into it. And than set it to Load. Is that what you are asking for?
```
# Create new coordinate system
clientObject = model.clientModel.factory.create('ns0:coordinate_system')
clientObject.no = 2
clientObject.type = 'TYPE_2_POINTS_AND_ANGLE'
clientObject.origin_coordinate_x = 0
clientObject.origin_coordinate_y = 1
clientObject.origin_coordinate_z = 1
clientObject.uw_plane_angle = 0.17453292519943295
...
model.clientModel.service.set_coordinate_system(clientObject)
```

Pull request for Coordinate System Class was just created ([here](https://github.com/Dlubal-Software/RFEM_Python_Client/pull/277/files)).
